src/main/analytics/service.ts

```diff
@@ -273,6 +273,19 @@ export function trackWebdavEvent(
 }
 
 export function trackWebdavError(
+  event: TrackedWebdavServerErrorEvents,
+  properties: Record<string, any>
+) {
+  const { uuid: userId } = ConfigStore.get('userData');
+  client.track({
+    userId,
```

Comment:
Can you check this? I believe something got ignored during the rebase, this will overwrite a fix I pushed some days ago, here is already in master-erb https://github.com/internxt/drive-desktop/blob/master-erb/src/main/analytics/service.ts#L290

Here is the PR https://github.com/internxt/drive-desktop/pull/279